ui/admin/tests/e2e/README.md

```diff
@@ -117,7 +118,7 @@ Using Terminal 1:
 - `$ cd boundary/enos`.
 - `$ doormat login`. Login with Doormat.
 - `$ eval "$(doormat aws export --account boundary_team_acctest_dev)"`. Exporting AWS env variables from doormat to your terminal.
-- `$ enos scenario launch e2e_ui builder:local`. Launch enos scenario, this will take from 5 to 10 minutes. When its done, you will see a Enos Operations finished! within your terminal.
+- `$ enos scenario launch e2e_ui_aws builder:local`. Launch enos scenario, this will take from 5 to 10 minutes. When its done, you will see a Enos Operations finished! within your terminal.
```

Comment:
ty for updating this- can you also take care of the destroy command on L155

Reply:
ah great catch, thank you!

Reply:
Good catch!! this might suffer changes with the pass of time. Also if you want to add a link to the available scenarios (plenty of them) https://github.com/hashicorp/boundary/tree/main/enos every `.hcl` is a scenario :wink: